reference-implementation/lib/writable-stream.js

```diff
@@ -747,6 +747,21 @@ function WritableStreamDefaultControllerClose(controller) {
   WritableStreamDefaultControllerAdvanceQueueIfNeeded(controller);
 }
 
+function WritableStreamDefaultControllerGetChunkSize(controller, chunk) {
```

Comment:
Originally I didn't inline it because it was a controller operation rather than a writer operation.

Afterwards I realised that once we have a ByteController this will become a polymorphic method call. Maybe the "you ain't gonna need it" principle applies, but I'm pretty sure that we will add a byte controller and that this separation will make sense at that time.